usb_product_ids.psv

```diff
@@ -292,7 +292,7 @@ Vendor ID | Product ID | Description
 0x1d50 | 0x6143 | [https://git.osmocom.org/osmo-small-hardware/tree/clock-generator Osmocom clock generator (DFU)]
 0x1d50 | 0x6144 | [https://osmocom.org/projects/e1-t1-adapter/wiki/ICE40_E1_USB_interface Osmocom E1 adapter (DFU)]
 0x1d50 | 0x6145 | [https://osmocom.org/projects/e1-t1-adapter/wiki/ICE40_E1_USB_interface Osmocom E1 adapter]
-0x1d50 | 0x6146 | [https://github.com/smunaut/ice40-playground Generic iCE40 DFU bootloader]
+0x1d50 | 0x6146 | [https://github.com/no2fpga/no2bootloader Nitro FGPA bootloader]
```

Comment:
`FGPA bootloader` should be `FPGA bootloader` I guess?

Reply:
thatnks for the review. fixed in 4121502cb81ac3fe630dd4625ed3b4c8b54e7406